test/services/ReviewServiceSpec.scala

```diff
@@ -588,6 +639,24 @@ class ReviewServiceSpec extends UnitSpec with MockFactory with ReviewData with A
           }
         }
       }
+      "the updateReview fails to update the details" should {
```

Comment:
Does updateReview return NotFound. It looks like it returns DatabaseError to me.

Reply:
I think we might have to look at all repo code in general for insert or updates, a lot of code just checks for Right(_) and lets anything else fall through to the .recover section.
So yes, it does retun db error as not checking anything else,